rtl_433-next/build.json

```
@@ -1,10 +1,10 @@
 {
   "build_from": {
-    "aarch64": "homeassistant/aarch64-base:3.18",
-    "amd64": "homeassistant/amd64-base:3.18",
-    "armhf": "homeassistant/armhf-base:3.18",
-    "armv7": "homeassistant/armv7-base:3.18",
-    "i386": "homeassistant/i386-base:3.18"
+    "aarch64": "homeassistant/aarch64-base:3.19",
+    "amd64": "homeassistant/amd64-base:3.19",
+    "armhf": "homeassistant/armhf-base:3.19",
+    "armv7": "homeassistant/armv7-base:3.19",
+    "i386": "homeassistant/i386-base:3.19"
```

They stopped publishing base images to docker hub. I think this will get things running:

```suggestion
    "aarch64": "ghcr.io/home-assistant/aarch64-base:3.19",
    "amd64": "ghcr.io/home-assistant/amd64-base:3.19",
    "armhf": "ghcr.io/home-assistant/armhf-base:3.19",
    "armv7": "ghcr.io/home-assistant/armv7-base:3.19",
    "i386": "ghcr.io/home-assistant/i386-base:3.19"
```